src/components/Content/Content.tsx

```diff
@@ -174,50 +111,52 @@ const Content = (props: ContentProps): ReactElement => {
 		)
 	};
 
+	/**
+	 * Generates the sections that will be iterated to render the fields
+	 * as they're positioned. If no sections are provided it will return
+	 * a one-single column section using the field's name or colum if defined.
+	 * @returns the sections that contains the names of the fields
+	 */
+	const sectionsToRender = useMemo(() => {
+		const newSections = []
+
+		if (!sections) {
+			fields.forEach(field => {
+				const fieldName = field?.column ? field.column : field.name;
+				newSections.push([[ fieldName ]])
+			});
+
+			return newSections;
+		}
+
+		return sections;
+	}, [sections]);
+
 	return (
 		<MainWrapper>
 			<TitleWrapper>
 				<Title>{title}</Title>
-				<ButtonsWrapper canShowMore={canShowMore && values}>
-					{!values
-						? onEdit && (
-							<Button
-								color="teal"
-								variant="icon"
-								mIcon={AddIcon}
-								onClick={onEdit}
-							></Button>
-						)
-						: onEdit && (
-							<Button
-								color="gray"
-								variant="icon"
-								mIcon={EditIcon}
-								onClick={onEdit}
-							></Button>
-						)}
-					{canShowMore && values && (
-						<Button
-							color="teal"
-							variant="text"
-							label={showMore ? "Less Details" : "More Details"}
-							onClick={showDetails}
-						></Button>
-					)}
+				<ButtonsWrapper>
+					{buttons.map((button, idx) => (
```

Comment:
Here we're assuming buttons will always exist, but since they're optional this might throw an "can't map undefined" error. Also I believe the <ButtonsWrapper> should only render if buttons exist.